courses/sync_external_courses/external_course_sync_api_test.py

```diff
@@ -226,12 +226,17 @@ def test_create_or_update_external_course_page(  # noqa: PLR0913
         else:
             external_course_page.unpublish()
 
+    keymap = (
+        EmeritusKeyMap()
+        if external_course_data["course_run_code"].startswith("MO")
+        else GlobalAlumniKeyMap()
+    )
```

Comment:
Sorry, I should have communicated it earlier, I think a test UTIL would be great to avoid the repetition, We can pass it the run code and it will return us the name and the keymap, We can use keymap where required and platform when required.